go/src/koding/kites/kloud/provider/google/stack.go

```diff
@@ -254,6 +257,10 @@ func (s *Stack) ApplyTemplate(c *stack.Credential) (*stack.Template, error) {
 
 	t.Resource["google_compute_instance"] = resource.GCInstance
 
+	t.Variable["koding_stack_id"] = map[string]interface{}{
```

Comment:
> @ppknap instead of disabling what about adding the stack_id at the end of it automatically?

We need to do this for all possible resource types in GCP. I want to make this explicit in order to let user decide.